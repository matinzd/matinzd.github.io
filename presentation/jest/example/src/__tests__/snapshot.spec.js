const sample = require('../data/sample.json')

it('sample json should match snapshot', () => {
    expect(sample).toMatchSnapshot()
})