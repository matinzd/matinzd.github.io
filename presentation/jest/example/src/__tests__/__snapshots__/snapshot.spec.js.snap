// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample json should match snapshot 1`] = `
Object {
  "title": "This is snapshot testing example",
}
`;
